trappy/plotter/Constraint.py

```diff
@@ -276,6 +276,13 @@ def _expand(self):
                    Len[columns] != 1
                    Len[templates] != 1
             )
+
+            TODO  is the following correct?
+            Permute(
+                   Len[traces] != 1
+                   Len[columns] = 1
```

Comment:
Actually, shouldn't `templates` have length 1 here? I.e. you have multiple traces; if they're DataFrames that all have a certain column then `columns` should have length 1, and if they're FTraces that all have a certain event then `templates` should have length 1?